samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/FlipViewSamplePage.xaml

```diff
@@ -0,0 +1,94 @@
+﻿<Page x:Class="Uno.Toolkit.Samples.Content.Controls.FlipViewSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:local="using:Uno.Toolkit.Samples.Content.Controls"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:msui="using:Microsoft.UI.Xaml.Controls"
+	  xmlns:sample="using:Uno.Toolkit.Samples"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+	  mc:Ignorable="d">
+
+	<sample:SamplePageLayout x:Name="SamplePageLayout" IsDesignAgnostic="True">
+		<sample:SamplePageLayout.DesignAgnosticTemplate>
+			<DataTemplate>
+				<StackPanel Margin="0,20" Spacing="20">
+					<StackPanel.Resources>
+						<ResourceDictionary>
+							<Style TargetType="TextBlock">
+								<Setter Property="HorizontalAlignment" Value="Center" />
+								<Setter Property="TextWrapping" Value="Wrap" />
+							</Style>
+						</ResourceDictionary>
+					</StackPanel.Resources>
+
+					<TextBlock Text="Try to navigate throw the FlipView using the buttons inside it. " />
```

Comment:
```suggestion
					<TextBlock Text="Try to navigate the FlipView using the buttons inside it. " />
```
Or, did you mean "... using swipe gesture or the buttons inside it."?

Reply:
The buttons inside it